packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/KYCNotification/template.js

```diff
@@ -51,15 +51,15 @@ const KYCNotification = props => {
                   id='scenes.buysell.coinifycheckout.content.kycnotification.limitsnotice.sell'
                   defaultMessage='While your identity gets verified, you can sell up to '
                 />
-                {sellMax} BTC.
+                &nbsp;{sellMax} BTC.
```

Comment:
default message has a space in the end

Reply:
Good eye, but it looks like the space in the message isn't being rendered. Keeping the `&nbsp;` and removing the space from the message so there's no confusion.